Dockerfile

```diff
@@ -15,11 +15,10 @@ RUN mkdir .cargo \
     && cargo vendor > .cargo/config
 
 # Install required system dependencies and cleanup in the same layer
-RUN apt-get update && apt-get install -y pkg-config libssl-dev python g++ git make && apt-get clean && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y pkg-config libssl-dev python g++ git make libglib2.0-dev && apt-get clean && rm -rf /var/lib/apt/lists/*
 
 # Copy the rest of the source code
 COPY . .
-RUN sed -i 's/"cloudflare",\s*//;s/, "cloudflare"//g' Cargo.toml
```

Comment:
We shouldn't copy cloudflare btw.

Reply:
@tusharmath 
Was a redundant line and how no effect on image size when I was doing it with amit